src/practice/ts/Components/Store/Add.ts

```diff
@@ -0,0 +1,45 @@
+import Storage from "./storage";
+
+class Add {
```

Comment:
Add라는 네이밍이 살짝 애매한거 같아요. 코드 내용을 보면 history를 추가하니까 AddHistory는 어떨까요?
```suggestion
class AddHistory {
```

Reply:
AddHistory가 확실히 명확한 것 같네요. 수정 반영했습니다.